src/app/core/models/issue.model.ts

```diff
@@ -324,6 +324,41 @@ export enum STATUS {
   Done = 'Done'
 }
 
+export enum ATTRIBUTES {
+  Severity = 'severity',
+  Type = 'type',
+  Response = 'response',
+  ResponseTag = 'responseTag',
+  Status = 'status',
+  Undefined = 'undefined',
+  Others = 'others'
+} 
+
+export enum SEVERITY {
+  VeryLow = 'VeryLow',
+  Low = 'Low',
+  Medium = 'Medium',
+  High = 'High'
+}
+
+export enum BUG {
```

Comment:
Might want to rename this as `BUG_TYPE` to be consistent with the attribute enum above